Decos.Diagnostics.Trace/TraceSourceLogFactory.cs

```diff
@@ -84,8 +85,8 @@ private TraceSource CreateSource(SourceName name)
             var switchValue = Options.GetLogLevel(name).ToSourceLevels();
             var traceSource = new TraceSource(name, switchValue);
 
-            var listeners = Options.Listeners
-                .Concat(System.Diagnostics.Trace.Listeners.Cast<TraceListener>());
+            var listeners = Options.Listeners;
+                //.Concat(System.Diagnostics.Trace.Listeners.Cast<TraceListener>());
```

Comment:
Why did you remove this? This will prevent output from showing up in the Output Window in Visual Studio.

Reply:
I had a problem where the logs were sent twice because every listener existed twice.
That problem is solved however I forgot to undo this change 😅 Sorry about this one.
https://github.com/DecosInformationSolutions/Diagnostics/pull/6/commits/acfbe29825ad6982c091891b8abef92f3b91f8ae